### DISCUSSION TITLE
Move index.html and other .html files outside of the /src & /dist folders

### DISCUSSION THREAD

**Discussion by Probablybest** [WeAreAthlon/frontend-webpack-boilerplate#205] [Q&A] [answered]
The current setup is that all the files from within /src are essentially copied over into /dist. However I've been trying to amend the setup so that the new structure would be:

```
index.html
other.html
favicon.ico
|- src
   |-fonts
   |- images
   |- js
   |- scss
   |- videos
|- dist/assets
   |- css
   |- images
   |- js
   |- videos
```

When I try to update the config though I can't get the setup to watch both the .html files and compile the css/js/images etc into the dist folder whilst still watching for file changes. 

I've tried setting up different path structures like below to test but it never works properly.

```
path.join(__dirname, "build"),
path.join(__dirname, "scripts"),
```

**Comment by pnikolov** [comment, marked as answer]
@Probablybest, in order to modify your copy of the setup to do the required you need to do the following modifications:

`webpack.config.js` change this line:
```js
 const templateFiles = fs.readdirSync(environment.paths.source)
```
to this:
```js
 const templateFiles = fs.readdirSync(environment.paths.source + '/../')
```

Then change `configuration/environment.js` paths.output to:
```js
  output: path.resolve(__dirname, '../dist/assets/'),
```

And finally move your `.html` files one level above the `src` directory as you have listed in your issue's description.

I am attaching a git `.patch` file if you want to apply all the above changes at once:
[probablybest.patch](https://github.com/WeAreAthlon/frontend-webpack-boilerplate/files/12913749/probablybest.patch)
